frontend/src/utils/search/convertSearchParamsToProperTypes.ts

```diff
@@ -30,7 +31,18 @@ export function convertSearchParamsToProperTypes(
 }
 
 // Helper function to convert query parameters to set
-function paramToSet(param: string | string[] | undefined): Set<string> {
+function paramToSet(
+  param: string | string[] | undefined,
+  type?: string,
+): Set<string> {
+  if (type === "status") {
+    if (param === SEARCH_NO_STATUS_VALUE) {
+      param = "";
+    } else {
+      param = param ?? "forecasted,posted";
```

Comment:
always wary of being too comment heavy, but since this function says `Helper function to convert query parameters to set` and it's now doing quite a bit more than that, it probably is worth updating or adding a new comment

Reply:
Updated, good catch.